frontend/src/utilities/calendar.ts

```diff
@@ -281,7 +281,7 @@ export function getCalendarEvents(
             `RRULE:FREQ=WEEKLY;BYDAY=${byDay};UNTIL=${endRepeat}Z`,
             `EXDATE;TZID=America/New_York:${exDate}`,
           ],
-          description: c.title,
+          description: `Course: ${c.title}\nInstructor: ${c.professor_names.join(', ')}`,
```

Comment:
```suggestion
          description: `${c.title}\nBy ${c.professor_names.join(', ')}`,
```

What do you think?

Reply:
Considered doing this or "with <instructor>, but there are some edge cases where merging the two would sound weird (i.e. if the course name has "by", "with", etc.)

Reply:
Yeah, but the line break would fix the issue with ambiguity, I hope. Otherwise "Instructors" is also fine, I just want to get rid of the "Course" part